dom-manipulation/dom-manipulation-quiz-notes.md

```diff
@@ -0,0 +1,55 @@
+# dom-manipulation-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `className` property of element objects?
+it's the class name of the html element.
+
+- How do you update the CSS class attribute of an element using JavaScript?
+by using  .className
+
+- What is the `textContent` property of element objects?
+to access the text content inside that element
+
+- How do you update the text within an element using JavaScript?
+set .textContent to a new contet
+
+- Is the `event` parameter of an event listener callback always useful?
+yes?
```

Comment:
Take a look at the function `countClicks()`. Notice that you didn't use the `event` parameter.